cli/src/config/templates.toml

```diff
@@ -123,13 +123,20 @@ separate(" ",
 
 description_placeholder = '''
   label(if(empty, "empty ") ++ "description placeholder", "(no description set)")'''
+email_placeholder = 'label("email placeholder", "(no email available)")'
```

Comment:
Thoughts on calling these "no _ set" to match the description placeholder?

Reply:
Done `s/available/set/`. I think both works, and shorter and consistent output is better.